src/main/java/persistence/sql/dml/SelectQuery.java

```diff
@@ -0,0 +1,49 @@
+package persistence.sql.dml;
+
+import persistence.sql.dml.clause.WhereClause;
+import persistence.sql.dml.clause.operator.WhereClauseSQLBuilder;
+
+import java.util.Arrays;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class SelectQuery {
+    public static final String ALL_SELECT_COLUMNS = "*";
+    private Set<String> selectClause;
+    private WhereClause whereClause;
```

Comment:
```suggestion
    public static final String ALL_SELECT_COLUMNS = "*";

    private Set<String> selectClause;
    private WhereClause whereClause;
```
상수와 상태값을 구분 지어주는 것도 좋습니다 :)